manifests/kots-config.yaml

```diff
@@ -720,58 +719,68 @@ spec:
           type: bool
           default: "0"
 
-        ## Imports / Exports
-        - name: datawarehousesExportsImportsEnabled
-          title: Imports & Exports in Datawarehouses
+        # Bigquery exports
+        - name: datawarehousesExportsBigQueryEnabled
+          title: Enable exports from BigQuery
           type: bool
           default: "0"
           help_text: |-
-            Enable when you are working with Redshift, RDS or Snowflake. More information available in our [documentation](https://docs.carto.com/~/changes/1Lia4Pv0PjH33F9R0CyP/carto-self-hosted/guides/configure-your-own-buckets).
-        - name: datawarehousesExportsImports
-          type: heading
-          title: Imports & Exports in Datawarehouses
-          when: '{{repl (ConfigOptionEquals "datawarehousesExportsImportsEnabled" "1") }}'
+            Enable exporting data from BigQuery connections.
         - name: storageBucketsExportGcp
-          title: GCP exports bucket name
-          help_text: The bucket name to store exported data from BigQuery
+          title: BigQuery exports bucket name
+          help_text: The bucket name to store exported data from BigQuery. If you need further info, please check [docs.carto.com](https://docs.carto.com/~/changes/1Lia4Pv0PjH33F9R0CyP/carto-self-hosted/guides/configure-your-own-buckets#configure-data-export-bucket-for-bigquery).
```

Comment:
In the next sections, you added the link at the same level that you enable it but here you added in a field.
I think that this should be writted also on the `datawarehousesExportsBigQueryEnabled` field, right? :thinking:

Reply:
Not really to be honest 🤔 The next section needs more explanation because the config is more complex. That's why I added it at the top level. On this one, if the customer reads `Enable exports from BigQuery and CARTO Data Warehouse` it's easy to understand it. Once they enable it, then we show that text field with the needed link to the docs if needed, but it's not complex configuration. What do you think? 😄